### PR TITLE
Add pause and resume button for triggers and jobs

### PR DESCRIPTION
**Pull request by mplayton**
You can now pause and resume a trigger from the UI. The pause/resume button will only show up for a job if there are one or more cron triggers for that job.

### CODE REVIEW

**Comment by disaacson** [comment]
If you have it up, mind sharing a screenshot?

**Comment by mplayton** [comment]
piezo-trigger
resume-job

File: admin/app/com/lucidchart/piezo/admin/controllers/Triggers.scala [hunk 1]
```diff
@@ -139,6 +140,8 @@ class Triggers(schedulerFactory: WorkerSchedulerFactory) extends Controller {
     }
   }
 
+
```
Comment (bravefoot) [inline admin/app/com/lucidchart/piezo/admin/controllers/Triggers.scala]: unnecessary whitespace

File: admin/app/com/lucidchart/piezo/admin/controllers/Jobs.scala [hunk 2]
```diff
@@ -68,9 +68,12 @@ class Jobs(schedulerFactory: WorkerSchedulerFactory) extends Controller {
           }
         }
 
-        val triggers = Some(scheduler.getTriggersOfJob(jobKey).asScala.toList)
+        val triggers = scheduler.getTriggersOfJob(jobKey).asScala.toList
+        val (resumableTriggers, pausableTriggers) = triggers.filter(_.isInstanceOf[CronTrigger]).map(_.getKey()).partition{ triggerKey =>
```
Comment (bravefoot) [inline admin/app/com/lucidchart/piezo/admin/controllers/Jobs.scala]: There's a lot going on in this line. You might want to split it before the map

File: admin/app/com/lucidchart/piezo/admin/controllers/Triggers.scala [hunk 3]
```diff
@@ -317,6 +320,42 @@ class Triggers(schedulerFactory: WorkerSchedulerFactory) extends Controller {
     }
   }
 
+  def pauseTrigger(group: String, name: String) = Action { implicit request =>
+    val jobKey = new TriggerKey(name, group)
```
Comment (bravefoot) [inline admin/app/com/lucidchart/piezo/admin/controllers/Triggers.scala]: Should be 'triggerKey', since you are pausing the trigger, rather than the job. Same thing in the next functon

File: admin/app/com/lucidchart/piezo/admin/views/job.scala.html [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
+pausableTriggers: List[TriggerKey] = Nil,
+resumableTriggers: List[TriggerKey] = Nil
 )(
 implicit
 request: play.api.mvc.Request[AnyContent]
 )
-@import com.lucidchart.piezo.admin.controllers.{routes=>piezoRoutes}
-@import com.lucidchart.piezo.admin.views
-@import java.net.URLEncoder
+
```
Comment (bravefoot) [inline admin/app/com/lucidchart/piezo/admin/views/job.scala.html]: whitespace

File: admin/app/com/lucidchart/piezo/admin/views/job.scala.html [hunk 5]
```diff
@@ -62,12 +75,13 @@ <h3 class="pull-left job-name">@currentJob.get.getKey.getGroup() &raquo; <span c
             var runSuccess = $('#runSuccess');
             var runError = $('#runError');
 
-            runConfirm.hide();
```
Comment (bravefoot) [inline admin/app/com/lucidchart/piezo/admin/views/job.scala.html]: :+1:

File: admin/app/com/lucidchart/piezo/admin/views/trigger.scala.html [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
 implicit
 request: play.api.mvc.Request[AnyContent]
 )
 
-@import com.lucidchart.piezo.TriggerMonitoringPriority
-@import com.lucidchart.piezo.admin.controllers.{routes=>piezoRoutes}
-@import com.lucidchart.piezo.admin.views
-@import java.net.URLEncoder
-@import org.quartz._
+
```
Comment (bravefoot) [inline admin/app/com/lucidchart/piezo/admin/views/trigger.scala.html]: Whitespace
